toc/ROLES.md

```diff
@@ -65,7 +79,10 @@ establish sub-working groups. Working groups delegate change approval to Approve
   <tr>
     <td>Approver</td>
     <td>
-        <p>Approve contributions from other members</p>
+        <p><b>For 1+ repos in an area:</b></p>
+        <p>Triage PRs and issues</p>
+        <p>Approve and merge contributions from other members</p>
+        <p>Perform small maintenance tasks, such as debugging test failures</p>
```

Comment:
```suggestion
        <p>Perform maintenance tasks, such as debugging test failures</p>
```

For me personally, I wouldn't need the entry at all. I see this as a logical part of the role as an approver. If we add it though (which I am also totally ok with), I wouldn't artificially limit this. It is going to involve _large_ maintenance tasks as well, and design work, and community interaction, and... .